testdata/google/events/cloud/scheduler/v1/SchedulerJobData-simple.json

```diff
@@ -0,0 +1,3 @@
+{
+  "customData": "my base64 data"
```

Comment:
I think it would be worth making this data that's obviously genuine base64. (This is valid base64, but with meaningless output.)
For example, "bXkgYmFzZTY0IGRhdGE=" is the base64 encoded version of "my base64 data".

Reply:
Yes. I need to motivate myself more. Thanks!